Ginger/GingerCore/Actions/ActDBValidation.cs

```diff
@@ -265,28 +265,48 @@ public override void Execute()
             }
         }
 
+        NoSqlBase.NoSqlBase NoSqlDriver = null;
         private void HandleNoSQLDBAction()
         {
-            NoSqlBase.NoSqlBase NoSqlDriver = null;
-
-            switch(this.DB.DBType)
+            //NoSqlDriver = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5368638)